### PR TITLE
doc updates

### PR DESCRIPTION
**Pull request by c-brenn**
- adds `features.md` as per customer's suggestion
- adds change log
- moves feature list to new doc
- moves service specific documentation to their own readmes
- adds test instructions for specific features using feature tags

Opening this for now - have to head out for a while, feel free to add more documentation.
Can prob squash this all into one commit when we're done.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+- PML upload. Added a form that allows users to upload a file. The file is sent to Panacea for analysis.
+- PML error & warning highlights. The results of file analysis are now reported back to the user.
+- Asclepius DDI endpoint. Asclepius now exposes an endpoint that takes a list of
+  drugs and returns any DDIs between those drugs.
+
+### Removed
+- pathways. The old docker service that
+  contained [peos](https://github.com/jnoll/peos) has been removed. It is
+  replaced by Asclepius and Panacea.
+- submodules. The
```
Comment (houli) [inline CHANGELOG.md]: What is this supposed to say?

Comment (c-brenn) [reply CHANGELOG.md]: oopsie. supposed to say that we removed the submodules and do everything in docker build

File: FEATURES.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+Once a file has been selected, users must be able to load it into the system for analysis.
+
+### Testing
+This feature has automated tests which can be run with the following command
+
+```bash
+$ docker run -e "MIX_ENV=test" tomtoothfairies/panacea mix test --only pml_loading
+```
+
+TODO: add tags and generic run instructions
```
Comment (houli) [inline FEATURES.md]: Is this still supposed to be here?

Comment (c-brenn) [reply FEATURES.md]: 🚫

**Comment by c-brenn** [comment]
I should really be putting `[skip ci]` in these commits

**Comment by c-brenn** [comment]
reminder to change `[Unreleased]` to `[0.3]` before merging

**Review by houli — APPROVED** [review]

**Pull request merged**